eng/common/TestResources/New-TestResources.ps1

```diff
@@ -574,15 +576,15 @@ try {
             }
         }
 
-        $postDeploymentScript = $templateFile | Split-Path | Join-Path -ChildPath 'test-resources-post.ps1'
+        $postDeploymentScript = $templateFile.originalFilePath | Split-Path | Join-Path -ChildPath 'test-resources-post.ps1'
         if (Test-Path $postDeploymentScript) {
             Log "Invoking post-deployment script '$postDeploymentScript'"
             &$postDeploymentScript -ResourceGroupName $ResourceGroupName -DeploymentOutputs $deploymentOutputs @PSBoundParameters
         }
 
-        if ($templateFile.EndsWith('.compiled.json')) {
-            Write-Verbose "Removing compiled bicep file $templateFile"
-            Remove-Item $templateFile
+        if ($templateFile.jsonFilePath.EndsWith('.compiled.json')) {
```

Comment:
I don't remember from the original PR to add bicep support, but I think this should go into $env:TEMP and we don't really have to worry about it - not in the repo. No worries about it getting checked in then either.

Reply:
It does generate to the temp directory but I'd like to try and make an attempt to clean it up if possible as I don't like an ever growing temp directory.

Reply:
Fair enough, but you should turn out temporary %TEMP% file clean up in Windows. In Settings, search for "Delete temporary files".